modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -5,7 +5,7 @@
 [id="op-interacting-with-pipelines-using-the-developer-perspective_{context}"]
 = Interacting with Pipelines using the Developer perspective
 
-The *Pipelines* view in the *Developer* perspective lists all the Pipelines in a project along with details, such as the namespace in which the Pipeline was created, the last PipelineRun, the status of the Tasks in the PipelineRun, the status of the PipelineRun, and the time taken for the run.
+The *Pipelines* view in the *Developer* perspective lists all the pipelines in a project along with details, such as the namespace in which the pipeline was created, the last PipelineRun, the status of the Tasks in the PipelineRun, the status of the PipelineRun, and the time taken for the run.
```

Comment:
Just a suggestion to make a bullet list of the details.

Reply:
Done.